### ISSUE TITLE
Locking with Win+L results in L being "stuck"

### ISSUE THREAD

**Issue by HaxtonFale** [damienleroy/PowerAccent#53]
Hi,
I often lock my machine with Win+L key combination and PowerAccent seems convinced I'm holding the L key when I unlock (e.g. space gets intercepted as the shortcut for accents on L instead of allowing me start unlocking the machine).

**Comment by damienleroy** [comment]
Hello,
Normally it should be intercept only with the space bar. Can you tell me if you are using the last version?

**Comment by damienleroy** [comment]
FYI, I can't reproduce it on my side :(

**Comment by HaxtonFale** [comment]
Hi, sorry for the delayed reply.
I was using the latest version, I think. I realised it was clashing with PowerToys one so I removed it but I'll try installing the current latest and see if it happens again.
I also have a similar problem with discord on this machine.

**Comment by HaxtonFale** [comment]
Okay, installed the latest version and reproduced the issue:
1. Lock the PC with Win + L
2. Hit Space to pull up the password prompt (and fail)
3. Unlock the machine
4. There should be a prompt for accents on L
If you can't reproduce it in this way, then perhaps the real issue is that I have something else blocking my keystrokes. I will also attempt to do this on another machine later today.

**Comment by HaxtonFale** [comment]
Confirmed on a co-worker's PC. Still need to check on my home PC to check if it's something about our work setups.

**Comment by HaxtonFale** [comment]
On my home PC it's not an issue, oddly enough. I can lock my PC with Win+L and then get to the password prompt by hitting space.

**Comment by damienleroy** [comment]
Ok, I have been able to reproduce it. And have an explanation about of it is happenning.

Here what's happen:
- You key stroke `L` letter. Tool detect it and waiting for Space bar
- Lock screen appear, and so you release `L` letter. But the event is not catched by the tool (we don't know why yet).
- Space bar is stroke, so the tool detect the combo is done, and so display the selector.

I though it was because we are in the lock screen context, but space bar is well detected. So something is wrong when passing from normal screen to lock screen I guess.

**Comment by guido-visser** [comment]
Maybe this helps as well, I use virtual desktops a lot and have the arrow keys as triggers.

1. Lock with `Win` + `L`
2. Unlock
3. Try to change virtual desktops right after login using: `Ctrl` + `Win` + `Arrow Key`

**Comment by AJman14** [comment]
I've noticed a similar - if not the same - issue here.

The `L` accents will occasionally pop up when I move a window to or from my secondary monitor using `Shift` + `Win` + `Right`/`Left`. I'm not even pressing `L`. My activation keys are the arrows.

I have to press a non-function key (can't remember; maybe only `L` works?) to make the dialog disappear.

PowerToys `v0.63.0`

**Comment by AJman14** [comment]
> 1. Lock with `Win` + `L`
> 2. Unlock
> 3. Try to change virtual desktops right after login using: `Ctrl` + `Win` + `Arrow Key`

The steps to reproduce are even more simple than that. In step 3, simply press an arrow key.

**Comment by lmoustak** [comment]
Another way to trigger this:
1. Lock PC via <kbd>Win</kbd> + <kbd>L</kbd>
2. Press one the numpad keys to prompt an umlock
3. Unlock with PIN
4. Press the trigger button (spacebar or arrow keys) while focusing on a text input

The quick accent flyout will pop up for the letter L.

**Comment by AJman14** [comment]
Also affects `v0.64.0`

**Comment by damienleroy** [comment]
Yes. I've got a fix for that: https://github.com/damienleroy/PowerAccent/pull/78
And plan to release the fix in the week.
For PowerToys I will push the fix asap.
